config/locales/en.yml

```diff
@@ -999,6 +999,8 @@ en:
               not_in_chargeable_state: "Invoice %{id} is not in chargeable state!"
             base:
               charging_failed: 'Failed to charge the credit card.'
+            period:
+              invalid_range: 'must be between the account creation date and 12 months from now'
```

Comment:
Yeah, I don't know... There seems to be a mix of uppercase and lowercase messages for attributes validation :shrug: 

I think it's preferred to be in lowercase, because it may be concatenated with the attribute name (e.g. "Period must be .... ") but I'm not sure